chromium_src/chrome/browser/profiles/profile_window.cc

```diff
@@ -12,17 +12,13 @@
 #include "base/bind.h"
 #include "base/threading/sequenced_task_runner_handle.h"
 #include "brave/browser/profiles/brave_profile_manager.h"
+#include "brave/browser/profiles/profile_util.h"
```

Comment:
Needed?

Reply:
It was added by accident. It's fixed now, thx